docs/REST/throttling.md

```diff
@@ -5,9 +5,13 @@ Configuration stores have limits on the requests that they may serve. Any reques
 Throttling is divided into different quota policies:
```

Comment:
We could use a numbered list, bullet list or a table to organize these four quota policies.

Reply:
We don't want to list all policies here specifically. There are several short-term throttling policies are not listed. Here we explained quota we calculated for throttling

Reply:
Yes, my comment was to make them numbered such as the following, or use bullet points.

1. **TotalRequests** - total amount of requests/sec
1. **WriteRequests** - total amount of write requests/second
1. **Bandwidth** - outbound data in bytes
1. **Storage** - total storage size of user data in bytes

Reply:
I don't mind either way. We have a minimilistic approach in the REST docs and to me the new lines are fine without bullets. If you all do decide to do bullets, please don't number them. There is no order associated with these quotas.

Reply:
The different quota policies are basically list items, so it makes sense to structure them that way. I'm fine with bullets, similar to how the operations are structured in our other document.

https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md